src/test/java/cloud/prefab/client/integration/DynamicTestBuilder.java

```diff
@@ -0,0 +1,76 @@
+package cloud.prefab.client.integration;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import cloud.prefab.client.Options;
+import cloud.prefab.client.PrefabCloudClient;
+import cloud.prefab.client.integration.Expectation;
+import cloud.prefab.client.integration.Scenario;
+import cloud.prefab.domain.Prefab;
+import java.util.Map;
+import java.util.Optional;
+import org.junit.jupiter.api.DynamicTest;
+import org.junit.jupiter.api.function.Executable;
+
+public class DynamicTestBuilder {
+
```

Comment:
A comment w/ the format of the yaml would make for easier reading

Reply:
Done!